src/scenes/Projects/Overview/ProjectOverview.tsx

```diff
@@ -100,14 +103,20 @@ export const ProjectOverview: FC = () => {
     },
   });
 
+  const { data: engagementListData } = useProjectEngagementListOverviewQuery({
+    variables: {
+      input: projectId,
+    },
+  });
```

Comment:
What's to stop this from starting to run while the main query is still fetching?

Reply:
There's nothing stopping it, it's an independent query that runs automatically on component mount.  Is the concern here that this query finishes before the project details?  If so we can render the engagement list if and only if the project details main query finishes also

Reply:
Well, I guess you would know more since you work on the API side a lot more than I do: Is having the db/API work on two queries at once going to make the first query more likely to lag? If not, then it doesn't matter

Reply:
I would guess it's treated like separate requests? Like a rest api?? @CarsonF @michaelmarshall let me know if I'm off base with this assumption

But I did need to only load the engagement cards after the above data has loaded so I added that check.